src/main/java/org/wso2/carbon/mongodb/util/MongoDBRealmUtil.java

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.mongodb.util;
+
+import java.util.Map;
+
+import org.wso2.carbon.mongodb.user.store.mgt.MongoDBRealmConstants;
+import org.wso2.carbon.mongodb.user.store.mgt.caseinsensitive.MongoDBCaseInsensitiveConstants;
+
+/**
+ * Map default MongoDB User store queries if they are not configured in user-mgt.xml
+ */
+public class MongoDBRealmUtil {
+
+    /**
+     * Get MongoDB user store properties (Sets default properties if already not available)
+     *
+     * @param properties map with user store properties
+     * @return map with default properties added
+     */
+    public static Map<String, String> getMongoProperties(Map<String, String> properties) {
+
+        // Realm properties
+        if (!properties.containsKey(MongoDBRealmConstants.SELECT_USER)) {
```

Comment:
can we use switch-case here?

Reply:
Followed the same convention used in the inbuilt user stores in product IS. Hence, used if statements.